### ISSUE TITLE
Consider log-level when cannot reopen zk logfile

### ISSUE THREAD

**Issue by rimms** [jubatus/jubatus#1026]
When Jubatus cannot reopen zk logfile on SIGHUP signal, Jubatus prints this logs and shut down the process.

```
FATAL [signals.cpp:133] exception in sigwait thread: Dynamic exception type: jubatus::core::common::exception::runtime_error::what: cannot open zk logfile
```

I think this situation is NOT a critical situation that should stop the process.

**Comment by rimms** [comment]
I tested again:

```
$ jubaclassifier -z localhost:2181 -n test -l ~/zklog
$ chmod 000 ~/zklog/jubaclassifier.xxx.xxx.xx.xx_9199.zklog.24912
$ kill -s HUP 24912
2016-07-13 14:57:25,799 24913 INFO  [signals.cpp:121] caught SIGHUP
2016-07-13 14:57:25,827 24913 FATAL [signals.cpp:133] exception in sigwait thread: Dynamic exception type: jubatus::core::common::exception::runtime_error::what: cannot open zk logfile
    #0 [jubatus::core::common::exception::error_file_name_*] = /path/to/zklog/jubaclassifier.xxx.xxx.xx.xx_9199.zklog.24912
    #0 [jubatus::core::common::exception::error_errno_*] = Permission denied (13)
    #0 [jubatus::core::common::exception::error_api_func_*] = fopen
    #0 [jubatus::core::common::exception::error_at_file_*] = ../jubatus/server/common/zk.cpp
    #0 [jubatus::core::common::exception::error_at_line_*] = 129
    #0 [jubatus::core::common::exception::error_at_func_*] = virtual void jubatus::server::common::zk::reopen_logfile()
```

I will change to ERROR from FATAL.

**Comment by rimms** [comment]
I tested to ``log4cxx``.

* Cannot read ``log4cxx.xml``: print message to ``stderr`` and keep running (using previous setting) 

  ```
  $ jubaclassifier -g log4cxx.xml -f config/classifier/default.json
  $ chmod 000 log4cxx.xml
  $ kill -s HUP 16004
  2016-07-20 17:24:49,795 16005 INFO  [signals.cpp:121] caught SIGHUP
  2016-07-20 17:24:49,796 16005 INFO  [server_util.cpp:75] reloading log configuration: /path/to/log4cxx.xml
  log4cxx: Could not open file [/path/to/log4cxx.xml].
  ```

* Cannot open log-file specified in ``log4cxx.xml``: print message to ``stderr`` and keep running

  ```
  $ jubaclassifier -g log4cxx.xml -f config/classifier/default.json
  $ chmod 000 jubaclassifier.xx.xxx.xxx.xxx_9199.jubatus.log.16004
  $ kill -s HUP 16004
  2016-07-20 17:32:00,080 16005 INFO  [signals.cpp:121] caught SIGHUP
  2016-07-20 17:32:00,081 16005 INFO  [server_util.cpp:75] reloading log configuration: /path/to/log4cxx.xml
  log4cxx: setFile(jubaclassifier.10.217.199.104_9199.jubatus.log.16004,true) call failed.
  log4cxx: IO Exception : status code = 13
  ```

**Comment by kmaehashi** [comment]
Thank you! Fixed via #1133
